homework/shelkovich_ilya/product/src/test/java/com/tutrit/java/quickstart/service/ProductServiceTest.java

```diff
@@ -0,0 +1,27 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Product;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProductServiceTest {
+
+    Product product;
+    ProductService productService;
+
+    @Before
+    public void setUp() {
+        this.product = new Product("water");
+        this.productService = new ProductService();
+    }
+
+    @Test
+    public void actualPrice() {
+        double regularPrice = 10;
```

Comment:
That is property of a product. 

Basically you've created a service that calculate discounts based on price, not the Product discount. 

It could work, if you write a general multipurpose service, but in this case you should work with product.

Reply:
Woooaaah, now I understand. Thx!